dbHelper.js

```diff
@@ -0,0 +1,36 @@
+const db = require("./data/dbConfig");
+
+function getUsers() {
+  return db("users");
+}
+
+function getUser(userID) {
+  return db("users")
+    .where({ id: userID })
+    .first();
+}
+
+function register(newUser) {
+  console.log("%cnewuser", "color:blue; font-size:16px;", newUser);
+  // return db("users")
+  //   .where({ username: newUser.username })
+  //   .then(id => {
+  //     res.status(404).json({ message: `${res} username taken` });
+  //   })
+  //   .catch(() => {
+  //     return db("users").insert(newUser);
+  //   });
+
+  return db("users").insert(newUser);
+}
+
+function login(username) {
+  return db("users").where({ username: username });
+}
+
+module.exports = {
```

Comment:
if you put this at the top of the file, when you or a future dev comes to the file and looks at it, they will instantly see what is being exported instead of having to scroll to the bottom to see all the methods